### PR TITLE
Code Changes

### PR DESCRIPTION
**Pull request by ParasuramanA**
1. Sidebar changed.
2. Assessment and task module notification added.
3. Home displaying two times fixed
4. Removed the required field in the submission link.
5. Batch list and task, assessment scrollable added.
6. Applicant view more page alignments fix also added dummy documents.

discussion:
Once the weightage branch code is merged in dev, I will write the code for the user assigned for this task. If the code is not merged in the dev branch I will complete the above functionality for user assigned for this task also.

### CODE REVIEW

File: src/pages/assessment/Assessment.jsx [hunk 1]
```diff
@@ -103,6 +109,8 @@ const AssessmentModule = () => {
               }
             )
             .then((res) => {
+              setEditId(null);
```
Comment (avinashdckap) [inline src/pages/assessment/Assessment.jsx]: Why we're setting two editId's? can't we use just one state

File: src/routes/PrivateRoute.jsx [hunk 2]
```diff
@@ -16,25 +16,17 @@ const PrivateRoute = () => {
...
-      subMenu: [
-        { label: "Task", id: "module/task" },
-        { label: "Assessment", id: "module/assessment" },
-        { label: "Quiz", id: "module/quiz" },
-      ],
-    },
-    { label: "Settings", id: "settings" },
+    { label: "Applications", id: "applications" },
+    {label:"Task",id :"module/task"},
+    {label:"Assessment",id: "module/assessment"},
```
Comment (avinashdckap) [inline src/routes/PrivateRoute.jsx]: for id just keep task  and assessment no need for module

File: src/pages/task/TaskModule.jsx [hunk 3]
```diff
@@ -71,22 +71,28 @@ const TaskModule = () => {
     };
 
     if (isDraft) {
-      Modal.warning({
-        title: "Success",
-        content: "Task Deleted Successfully...",
+      Modal.confirm({
```
Comment (avinashdckap) [inline src/pages/task/TaskModule.jsx]: For Task created / updated Succesfully are we using isDraft to change the notification message?

**Review by avinashdckap — CHANGES_REQUESTED** [review]

**Comment by avinashdckap** [comment]
You cant hard code the height


element.style {
}
<style>
body .container .listing-container .task-list-container {
    height: 460px;
    overflow: scroll;
}

like this , in my screen it shows only half tasks  add the height dynamically using calc or ask sugumar

**Comment by avinashdckap** [comment]
Scroll should be only shown only when items(tasks or assessment) exceeds the screen height , dont show it by default

**Comment by avinashdckap** [comment]
Still the delete modal on click showing multiple times

**Comment by avinashdckap** [comment]
add  back icon on back to dashboard

**Comment by avinashdckap** [comment]
There is no hover effect for menu items like we have tasks and assesment

File: src/pages/task/TaskModule.jsx (hunk 3 above)
Comment (ParasuramanA) [reply src/pages/task/TaskModule.jsx]: I have changed

File: src/routes/PrivateRoute.jsx (hunk 2 above)
Comment (ParasuramanA) [reply src/routes/PrivateRoute.jsx]: I have changed

File: src/pages/assessment/Assessment.jsx (hunk 1 above)
Comment (ParasuramanA) [reply src/pages/assessment/Assessment.jsx]: I have changed

**Pull request merged**